manager/controllers/app/blueprint_deploy_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+package app
+
+import (
+	"context"
+
+	app "github.com/mesh-for-data/mesh-for-data/manager/apis/app/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func deployBlueprint(namespace string, shouldSucceed bool) {
+	blueprint := &app.Blueprint{}
+	Expect(readObjectFromFile("../../testdata/blueprint-read.yaml", blueprint)).ToNot(HaveOccurred())
+
+	// Set the correct namespace
+	blueprint.SetNamespace(namespace)
+
+	blueprintKey := client.ObjectKeyFromObject(blueprint)
+
+	// Create Blueprint
+	Expect(k8sClient.Create(context.Background(), blueprint)).Should(Succeed())
+
+	// Ensure getting cleaned up after tests finish
+	defer func() {
+		blueprint := &app.Blueprint{ObjectMeta: metav1.ObjectMeta{Namespace: blueprintKey.Namespace, Name: blueprintKey.Name}}
+		_ = k8sClient.Get(context.Background(), blueprintKey, blueprint)
+		_ = k8sClient.Delete(context.Background(), blueprint)
+	}()
+
+	By("Expecting blueprint to be created")
+	Eventually(func() error {
+		return k8sClient.Get(context.Background(), blueprintKey, blueprint)
+	}, timeout, interval).Should(Succeed())
```

Comment:
I think that the declaration of timeout and interval as package scoped variables (in `m4dapplication_controller_test.go`) was not intentional and these should have been defined like in `batchtrasnfer_controller_test.go` instead.

Reply:
Fixed